src/packages/files/files.ts

```diff
@@ -89,6 +90,58 @@ export default (app: Express) => {
     }
   });
 
+  /**
+   * @swagger
+   * /file/{id}:
+   *   get:
+   *     parameters:
+   *       - description: ID of the file
+   *         in: path
+   *         name: id
+   *         required: true
+   *         schema:
+   *           type: number
+   *     produces:
+   *       - application/json
+   *     responses:
+   *       200:
+   *         description: Fetches the file by id
+   *     summary: Get the file with the associated ID.
+   *     tags:
+   *       - Files
+   */
+  app.get('/file/:id', filesLimiter, auth, async (req, res) => {
+    const { params } = req;
+    const { id } = params;
+
+    try {
+      const { rows, rowCount } = await db.query(
+        `
+        select *
+        from merlin.uploaded_file
+        where id = $1;
```

Comment:
You may want to add `and deleted_date is null` after `id = $1`, so that it doesn't return a file that was "deleted" (you'd have to ask @mattdailis why we keep files that the user has deleted, I'm blanking on the exact reasoning at the moment)